wurst/data/LinkedList.wurst

```diff
@@ -327,6 +329,43 @@ public interface LLItrClosure<T>
 public interface LinkedListUpdater<T>
 	function update(T t) returns T
 
+/*  String processing extra functions 
+	The only reason this is here is that it can't be in String.wurst due to dependency problems */
```

Comment:
Not a good enough reason. Is there a circular dependency?

Reply:
yes but I figured I will just put it in StringUtils and there's no problem